### PR TITLE
Installation added repositories

### PR DESCRIPTION
**Pull request by lbarcziova**
When creating #1583, I was looking into the related code and noticed we handle also events when repositories are being added to the app installation, but we were not updating the `repositories` in the DB  (until my changes I made in #1575, which were meant mainly for reinstallation to have the new `sender_login` and `created_at` data and the `repositories` is only rewritten by the added repos which is not correct). Also, we don't react to events when repositories are being removed from the app installation.  We can discuss whether we want to react to both `added` and `removed` events and actually update the `repositories` in the DB properly, but with the current codebase, I think it doesn't make sense to react to these, but maybe somebody can correct me what was the reason for handling these events.

---

RELEASE NOTES BEGIN
￼
RELEASE NOTES END

### CODE REVIEW

File: packit_service/worker/handlers/forges.py [hunk 1]
```diff
@@ -64,22 +64,24 @@ def run(self) -> TaskResults:
         match.
         :return: TaskResults
```
Comment (mfocko) [inline packit_service/worker/handlers/forges.py:65]:
```suggestion

        Returns:
            Result of the run task.
```

**Review by mfocko — COMMENTED** [review]
lgtm

**Review by TomasTomecek — APPROVED** [review]
I appreciate the detailed write-up in the PR description, could you please add that info into a commit description so we have it in git as well? Please don't be afraid to break down compound sentences into smaller ones - the opening one is really complex, I had to read it 3 times :D

LGTM changes - it feels like it would be a much bigger effort to have 100% clarity in which repos Packit is installed. That doesn't seem worth it since we actually care about repos that actively use Packit.

**Comment by mfocko** [comment]
> I appreciate the detailed write-up in the PR description, could you please add that info into a commit description so we have it in git as well? Please don't be afraid to break down compound sentences into smaller ones - the opening one is really complex, I had to read it 3 times :D

should be included in the merge commit :P

**Comment by TomasTomecek** [comment]
ok, if that's the case - the info will be in the merge commit by GitHub

**Comment by lbarcziova** [comment]
@TomasTomecek but you are right that it is more clear to have it directly in the commit containing the changes, so I updated it there as well...sorry for the long sentence, that's my weakness :D

**Review by jpopelka — APPROVED** [review]

**Pull request merged**